### PR TITLE
SONAR-7027 Fix for dev cockpit

### PR DESCRIPTION
**Pull request by julienlancelot**
Fix for https://github.com/SonarSource/sonar-dev-cockpit/pull/6

### CODE REVIEW

File: sonar-db/src/main/java/org/sonar/db/permission/PermissionRepository.java [hunk 1]
```diff
@@ -155,10 +155,15 @@ public void applyPermissionTemplate(DbSession session, String templateUuid, long
     }
   }
 
-  public void grantDefaultRoles(DbSession session, long componentId, String qualifier) {
-    ResourceDto resource = dbClient.resourceDao().selectResource(componentId, session);
-    String applicablePermissionTemplateKey = getApplicablePermissionTemplateKey(session, resource.getKey(), qualifier);
-    applyPermissionTemplate(session, applicablePermissionTemplateKey, componentId);
+  // TODO Add UT
```
Comment (SonarTech) [inline sonar-db/src/main/java/org/sonar/db/permission/PermissionRepository.java]: INFO Complete the task associated to this TODO comment. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)

**Comment by SonarTech** [comment]
SonarQube analysis reported 4 issues:
* MAJOR 3 major
* INFO 1 info

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'protocol' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'input' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol/input) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'output' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol/output) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Comment by sns-seb** [comment]
LGTM

**Comment by SonarTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR Add a 'package-info.java' file to document the 'protocol' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'input' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol/input) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)
* MAJOR Add a 'package-info.java' file to document the 'output' package (org.sonarsource.sonarqube:sonar-batch-protocol:target/generated-sources/protobuf/org/sonar/batch/protocol/output) [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1228)

**Pull request merged**